pkg/analyzer/parser.go

```diff
@@ -71,7 +73,9 @@ func (p *NSXConfigParser) RunParser() error {
 	}
 	p.getGroups() // get groups config
 	p.removeVMsWithoutGroups()
-	p.getDFW() // get distributed firewall config
+	if err := p.getDFW(); err != nil { // get distributed firewall config
+		return err
```

Comment:
do we crash over error or just ignore rule with error?
does error follow from invalid ip/cidr?

Reply:
1. the parser returns error 
2. yes

Reply:
if you think it is better to warn & ignore the path, I will change it - it will reduce some code

Reply:
changed
(I did not ignore the rule, only the specific IP)